apps/aeutil/lib/scientific.ex

```diff
@@ -6,6 +6,8 @@ defmodule Aeutil.Scientific do
 
   use Bitwise
 
+  @highest_target_int 0xFFFF000000000000000000000000000000000000000000000000000000000000
```

Comment:
Maybe get this value from a `get_highest_target_int()` func so we don't preload the value in memory?

Reply:
Is this for performance optimization? The variable would be needed for every block validation.